tests/cassettes/TestInsuranceAll.yaml

```diff
@@ -5,109 +5,116 @@ interactions:
-      at USPS Facility","description":null,"status":"in_transit","status_detail":"arrived_at_facility","datetime":"2022-02-05T02:59:13Z","source":"USPS","carrier_code":null,"tracking_location":{"object":"TrackingLocation","city":"COLUMBIA","state":"SC","country":null,"zip":"29201"}},{"object":"TrackingDetail","message":"Arrived
-      at Post Office","description":null,"status":"in_transit","status_detail":"arrived_at_facility","datetime":"2022-02-05T05:50:13Z","source":"USPS","carrier_code":null,"tracking_location":{"object":"TrackingLocation","city":"CHARLESTON","state":"SC","country":null,"zip":"29407"}},{"object":"TrackingDetail","message":"Sorting
-      Complete","description":null,"status":"in_transit","status_detail":"status_update","datetime":"2022-02-05T11:30:13Z","source":"USPS","carrier_code":null,"tracking_location":{"object":"TrackingLocation","city":"CHARLESTON","state":"SC","country":null,"zip":"29407"}},{"object":"TrackingDetail","message":"Out
-      for Delivery","description":null,"status":"out_for_delivery","status_detail":"out_for_delivery","datetime":"2022-02-05T11:40:13Z","source":"USPS","carrier_code":null,"tracking_location":{"object":"TrackingLocation","city":"CHARLESTON","state":"SC","country":null,"zip":"29407"}},{"object":"TrackingDetail","message":"Delivered","description":null,"status":"delivered","status_detail":"arrived_at_destination","datetime":"2022-02-05T16:32:13Z","source":"USPS","carrier_code":null,"tracking_location":{"object":"TrackingLocation","city":"CHARLESTON","state":"SC","country":null,"zip":"29407"}}],"carrier_detail":{"object":"CarrierDetail","service":"First-Class
-      Package Service","container_type":null,"est_delivery_date_local":null,"est_delivery_time_local":null,"origin_location":"HOUSTON
-      TX, 77001","origin_tracking_location":{"object":"TrackingLocation","city":"NORTH
-      HOUSTON","state":"TX","country":null,"zip":"77315"},"destination_location":"CHARLESTON
-      SC, 29401","destination_tracking_location":{"object":"TrackingLocation","city":"CHARLESTON","state":"SC","country":null,"zip":"29407"},"guaranteed_delivery_date":null,"alternate_identifier":null,"initial_delivery_attempt":"2022-02-05T16:32:13Z"},"finalized":true,"is_return":false,"public_url":"https://track.easypost.com/djE6dHJrXzNlODNlYmU2NGZiYTQ3YmRhMjdiZjYwODkzZTQ4NmI3","fees":[]},"tracking_code":"9400100109361109729572","fee":{"object":"Fee","type":"InsuranceFee","amount":"1.00000","charged":true,"refunded":false},"messages":["The
-      tracker entered the mailstream before insurance was purchased."],"created_at":"2022-03-03T02:38:12Z","updated_at":"2022-03-03T02:39:45Z"}],"has_more":true}'
+    body: '{"insurances":[{"amount":"100.00000","created_at":"2022-05-25T18:03:30Z","fee":{"amount":"1.00000","charged":true,"object":"Fee","refunded":false,"type":"InsuranceFee"},"from_address":{"carrier_facility":null,"city":"San
```

Comment:
Yeah, nested objects are definitely out of order.

Reply:
correct, this only works with non-array object string.

Reply:
Hmm... that's too bad. Thank you for looking into this. If it will only work for one and not the other, I'd prefer if we didn't introduce the complexity and simply kept them all the same (ordered alphabetically). I'll cry myself to sleep but we shouldn't spend more time on it.